### DISCUSSION TITLE
Ackowlege version change after automatic update

### DISCUSSION THREAD

**Discussion by monabuu** [Homebrew/discussions#4175] [Casks]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Is there a way to tell brew that a cask has auto updated and the version has changed so it doent's show up in 'brew outdated --greedy'?

For example I strongly prefer upgrading WhatsApp and Reaper via brew but Signal does well on it's own but i have trouble sifting through outdated for casks that actually are outdated.

### Relevant casks

-

**Comment by SMillerDev** [comment]
No, there is no way for brew to know about updates unless it does the update
